rest-client/pom.xml

```diff
@@ -154,6 +154,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
That's fine, I was referring to this, which includes debug information as well:
 <plugin> 
    <groupId>org.apache.maven.plugins</groupId> 
    <artifactId>maven-compiler-plugin</artifactId> 
    <configuration> 
        <compilerArgument>-g:source,lines,vars</compilerArgument>
    </configuration> 
</plugin>